.github/actions/list-variants/action.yml

```diff
@@ -8,17 +16,6 @@ outputs:
     description: Variants that should not run for aarch64
     value: ${{ steps.get-variants.outputs.aarch-enemies }}
 runs:
-  using: "composite"
-  steps:
-    - uses: actions/checkout@v3
-    - id: get-variants
-      name: Determine variants
-      shell: bash
-      run: |
-        cd variants
-        output="variants=$(ls -d */ | cut -d'/' -f 1 | grep -vE '^(shared|target)$' | jq -R -s -c 'split("\n")[:-1]')"
-        echo $output
-        echo $output >> $GITHUB_OUTPUT
-        output="aarch-enemies=$(ls -d */ | cut -d'/' -f 1 | grep -E '(^(metal|vmware)|\-dev$)' | jq -R -s -c 'split("\n")[:-1] | [ .[] | {"variant": ., "arch": "aarch64"}]')"
-        echo $output
-        echo $output >> $GITHUB_OUTPUT
+  using: "node20"
+  main: 'dist/index.js'
```

Comment:
Have you tried not compiling the app, and just do what [this example] does?. I would like us to avoid including any "compiled" javascript file since they tend to be gigantic.

[this example]: https://github.com/duckduckgo/privacy-configuration

Reply:
Yeah, the alternative is to include the `node_modules` directory, at least from what I understand. That would be much more than the compiled output that gets somewhat minified in `dist/`.